Penumbra/UI/AdvancedWindow/ResourceTreeViewer.cs

```diff
@@ -102,6 +109,40 @@ public void Draw()
         }
     }
 
+    private void DrawControls()
+    {
+        var yOffset = (ChangedItemDrawer.TypeFilterIconSize.Y - ImGui.GetFrameHeight()) / 2f;
+        ImGui.SetCursorPosY(ImGui.GetCursorPosY() + yOffset);
+
+        if (ImGui.Button("Refresh Character List"))
+            _task = RefreshCharacterList();
+
+        ImGui.SameLine();
+        ImGui.Dummy(ImGuiHelpers.ScaledVector2(20, 0));
+
+        using (var id = ImRaii.PushId("TreeCategoryFilter"))
+        {
+            var spacing        = ImGui.GetStyle().ItemInnerSpacing.X;
+            var categoryFilter = (uint)_categoryFilter;
+            foreach (var category in Enum.GetValues<TreeCategory>())
+            {
+                ImGui.SameLine(0.0f, spacing);
+                using var c = ImRaii.PushColor(ImGuiCol.CheckMark, CategoryColor(category).Value());
+                ImGui.CheckboxFlags($"##{category}", ref categoryFilter, (uint)category);
+                ImGuiUtil.HoverTooltip(CategoryFilterDescription(category));
+            }
+            _categoryFilter = (TreeCategory)categoryFilter;
+        }
+
+        ImGui.SameLine();
+        ImGui.Dummy(ImGuiHelpers.ScaledVector2(20, 0));
+
+        ImGui.SameLine();
+        _changedItemDrawer.DrawTypeFilter(ref _typeFilter, -yOffset);
+
+        ImGui.InputTextWithHint("##TreeNameFilter", "Filter by Character/Entity Name...", ref _nameFilter, 128);
```

Comment:
ImGui.SetNextItemWidth(ImGui.GetContentRegionAvail().X);

Reply:
nvm I'll change it myself: the category icons are overlapping with the ALL on minimum window width with default style, so I reduced the padding to the sides of the checkmarks a bit.